vm/actor/src/builtin/system/mod.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+mod state;
+
+pub use self::state::State;
+use crate::{assert_empty_params, empty_return, SYSTEM_ACTOR_ADDR};
+use address::Address;
+use ipld_blockstore::BlockStore;
+use num_derive::FromPrimitive;
+use num_traits::FromPrimitive;
+use runtime::{ActorCode, Runtime};
+use vm::{ExitCode, MethodNum, Serialized, METHOD_CONSTRUCTOR};
+
+/// Init actor methods available
+#[derive(FromPrimitive)]
+pub enum Method {
+    Constructor = METHOD_CONSTRUCTOR,
+}
+
+impl Method {
+    /// Converts a method number into an Method enum
```

Comment:
I realized I have this typo in some of the actors, I can go back and switch to a more useful comment after but: 
```suggestion
    /// Converts a method number into a Method enum
```